components/display/pcd8544.rst

```diff
@@ -0,0 +1,72 @@
+PCD 8544 Display (Nokia 5110/3310)
+==================================
+
+.. seo::
+    :description: Instructions for setting up PCD8544 display drivers.
+    :image: pcd8544.jpg
+
+.. _pcd8544:
+
+Usage
+-----
+
+The ``pcd8544`` display platform allows you to use
+PCD1325 (`Adafruit <https://www.adafruit.com/product/338>`__)
```

Comment:
Is this PCD1325 too?

Reply:
oh god this must be some typo, probably there is no such display :D